CI actions regularly fail on nzbget

<!-- READ THIS FIRST:
  - If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
  - Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/core/releases
  - Do not report issues for integrations if you are using custom components or integrations.
  - Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
  DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
-->
## The problem
<!-- 
  Describe the issue you are experiencing here to communicate to the
  maintainers. Tell us what you were trying to do and what happened.
-->
When adding Pull Requests, CI actions sometimes fail due to unrelated components. It seems that nzbget comes up regularly.
For example in this instance: https://github.com/home-assistant/core/runs/1193242518:
```txt
>           assert state.state == data[1]
E           AssertionError: assert '2020-10-01T12:06:39+00:00' == '2020-10-01T12:06:38+00:00'
```


## Environment
<!--
  Provide details about the versions you are using, which helps us to reproduce
  and find the issue quicker. Version information is found in the
  Home Assistant frontend: Configuration -> Info.
-->

- Home Assistant Core release with the issue: 
- Last working Home Assistant Core release (if known): 
- Operating environment (OS/Container/Supervised/Core): 
- Integration causing this issue: nzbget
- Link to integration documentation on our website: 

## Problem-relevant `configuration.yaml`
<!--
  An example configuration that caused the problem for you. Fill this out even
  if it seems unimportant to you. Please be sure to remove personal information
  like passwords, private URLs and other credentials.
-->

```yaml

```

## Traceback/Error logs
<!--
  If you come across any trace or error logs, please provide them.
-->

```txt
_____________________________ test_sensors[pyloop] _____________________________
[gw1] linux -- Python 3.7.1 /__w/core/core/venv/bin/python

hass = <homeassistant.core.HomeAssistant object at 0x7ff5296d6a20>
nzbget_api = <MagicMock name='NZBGetAPI' id='140690938752472'>

    async def test_sensors(hass, nzbget_api) -> None:
        """Test the creation and values of the sensors."""
        now = dt_util.utcnow().replace(microsecond=0)
        with patch("homeassistant.util.dt.utcnow", return_value=now):
            entry = await init_integration(hass)
    
        registry = await hass.helpers.entity_registry.async_get_registry()
    
        uptime = now - timedelta(seconds=600)
    
        sensors = {
            "article_cache": ("ArticleCacheMB", "64", DATA_MEGABYTES, None),
            "average_speed": (
                "AverageDownloadRate",
                "1.19",
                DATA_RATE_MEGABYTES_PER_SECOND,
                None,
            ),
            "download_paused": ("DownloadPaused", "False", None, None),
            "speed": ("DownloadRate", "2.38", DATA_RATE_MEGABYTES_PER_SECOND, None),
            "size": ("DownloadedSizeMB", "256", DATA_MEGABYTES, None),
            "disk_free": ("FreeDiskSpaceMB", "1024", DATA_MEGABYTES, None),
            "post_processing_jobs": ("PostJobCount", "2", "Jobs", None),
            "post_processing_paused": ("PostPaused", "False", None, None),
            "queue_size": ("RemainingSizeMB", "512", DATA_MEGABYTES, None),
            "uptime": ("UpTimeSec", uptime.isoformat(), None, DEVICE_CLASS_TIMESTAMP),
        }
    
        for (sensor_id, data) in sensors.items():
            entity_entry = registry.async_get(f"sensor.nzbgettest_{sensor_id}")
            assert entity_entry
            assert entity_entry.device_class == data[3]
            assert entity_entry.unique_id == f"{entry.entry_id}_{data[0]}"
    
            state = hass.states.get(f"sensor.nzbgettest_{sensor_id}")
            assert state
            assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == data[2]
>           assert state.state == data[1]
E           AssertionError: assert '2020-10-01T12:06:39+00:00' == '2020-10-01T12:06:38+00:00'
E             - 2020-10-01T12:06:38+00:00
E             ?                   ^
E             + 2020-10-01T12:06:39+00:00
E             ?                   ^

tests/components/nzbget/test_sensor.py:54: AssertionError
```

## Additional information

Another instance:
https://github.com/home-assistant/core/pull/40943/checks?check_run_id=1192660693
```txt
>           assert state.state == data[1]
E           AssertionError: assert '2020-10-01T09:49:58+00:00' == '2020-10-01T09:49:57+00:00'
```

--- a/tests/components/nzbget/test_sensor.py
+++ b/tests/components/nzbget/test_sensor.py
@@ -17,7 +17,7 @@ from tests.async_mock import patch
 async def test_sensors(hass, nzbget_api) -> None:
     """Test the creation and values of the sensors."""
     now = dt_util.utcnow().replace(microsecond=0)
-    with patch("homeassistant.util.dt.utcnow", return_value=now):
+    with patch("homeassistant.components.nzbget.sensor.utcnow", return_value=now):
         entry = await init_integration(hass)
 
     registry = await hass.helpers.entity_registry.async_get_registry()

